jquery.flot.absRelTime.js

```diff
@@ -19,6 +19,12 @@ xaxis: {
     timeformat: null, // format string to use
     twelveHourClock: false, // 12 or 24 time in time mode
     monthNames: null // list of names of months
+    formatString: "" // A string that allows a user to format the time display (see below for details)
```

Comment:
I probably should combind formatString into timeFormat. I expect it would then take the form of, as an example:

"%A: #d#T.SS" or "%r: #d#T.SS" (where in the latter case the #d part of the format would be ignored).

The double quotes thing is just me being inconsistent.